apps/sms/views/inbox/test/unit/inbox_test.js

```diff
@@ -1073,6 +1073,11 @@ suite('thread_list_ui', function() {
         assert.equal(
           li.querySelector('.body-text').textContent, 'from a draft'
         );
+
```

Comment:
Wondering do we need to verify the time here, since the code didn't mention the timestamp in this case.